SoftLayer/managers/registration.py

```diff
@@ -0,0 +1,29 @@
+"""
+    SoftLayer.subnet_registration
+    ~~~~~~~~~~~~~~~~~~~~~~~
+    Registration manager
```

Comment:
Change this line to "Regional Internet Registry (RIR) Manager" to make it more clear what it manages